### DISCUSSION TITLE
SSH keys 作成時と yarn install 時のエラー

### DISCUSSION THREAD

**Discussion by rie-oki-agn** [commit-mate/commit-mate.net#20] [1. How to join the commit-mate]
![image](https://user-images.githubusercontent.com/85418440/175261678-c58b19aa-1468-4828-bc9c-d4a2c8daafb0.png)

node.jsのバージョンを変更して解決

**Comment by Pomerican** [comment]
## 「The engine "node" is incompatible with this module」のエラーについて
- ```$ yarn```もしくは```$ yarn install```時に出現
- 原因は**nodeのバージョンが非互換だった**
```bash
$ node -v
v16.0.0
```

以下のコマンドでnodeをLTS(ちょうどよい最新版)にアップ↑
```zsh
nvm install --lts
```
これだけだと```yarn```が消えちゃうので再ダウンロード
```bash
npm install -g yarn
```

**解決！！...ですが**
---
いつnvm入れたんだっけ...って思いました。。
所感です。。。

**Reply by rie-oki-agn** [reply]
尊い🥺
